src/java.desktop/share/classes/javax/swing/border/BevelBorder.java

```diff
@@ -86,6 +86,12 @@ public BevelBorder(int bevelType) {
     /**
      * Creates a bevel border with the specified type, highlight and
      * shadow colors.
+     * The bevel outer highlight color and
+     * bevel inner highlight color will be derived from
+     * specified highlight color and
+     * bevel outer shadow color
+     * and bevel inner shadow color
+     * will be derived from specified shadow color.
```

Comment:
It mostly looks fine, but probably we may reformat it better than 55 chars per line.

Reply:
Done